### PR TITLE
HomeScreen Component

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description
Created HomeScreen component, added less file for styling, added a button to add a child

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [x] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/HomeScreen/HomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+              </button>
+            </Content>
+          </Layout>
+        </Layout>
+      </>
+    );
+  }
+}
+
+//   <Layout>
```
Comment (bramirez96) [inline src/components/pages/HomeScreen/HomeScreen.js]: Drop the comment block and it'll be perfect!

**Review by Nick-Ohman — DISMISSED** [review]
looks great

File: src/components/pages/HomeScreen/HomeScreen.js (hunk 1 above)
Comment (bramirez96) [reply src/components/pages/HomeScreen/HomeScreen.js]: Perfect!

**Review by bramirez96 — APPROVED** [review]

**Review by Nick-Ohman — APPROVED** [review]
after you deleted that block of commented code out, it looks great!

**Pull request merged**